src/naemon/macros.c

```diff
@@ -139,51 +140,56 @@ int get_raw_command_line_r(nagios_macros *mac, command *cmd_ptr, char *cmd, char
 	/* get the full command line */
 	*full_command = nm_strdup((cmd_ptr->command_line == NULL) ? "" : cmd_ptr->command_line);
 
-	/* XXX: Crazy indent */
-	/* get the command arguments */
-	if (cmd != NULL) {
+	if (cmd == NULL) {
+		log_debug_info(DEBUGL_COMMANDS | DEBUGL_CHECKS | DEBUGL_MACROS, 2, "Expanded Command Output: %s\n", *full_command);
```

Comment:
In this case, it means that there was no need to further expand the command? I.e for commands like `DISABLE_NOTIFICATIONS` where there are no args

Reply:
sorry, this is not external commands, but check commands... Similar point though.

Reply:
tbh, i didn't check where this is used, i just inverted the huge if clause to return early instead of having the hole function in a huge if. The functionality did not change at this point.
I should have put this in a separate commit.